packages/xorg_lib.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'

```

Comment:
This should have a description, homepage and version.  Take a look at `buildessential` for an example.  Everything else looks great, however.

Reply:
Fix it.